dashboard/src/pages/Dashboard/Dataset/DatasetStart.tsx

```diff
@@ -77,6 +80,54 @@ export const DatasetStart = () => {
     },
   );
 
+  const reloadChunkCount = () => {
+    const datasetId = currDatasetId();
+    if (!datasetId) {
+      console.error("Dataset ID is undefined.");
+      return;
+    }
+
+    void (async () => {
```

Comment:
This is a really good place to use the Promise API instead of the async one. 

Before merging, this should get switched to using a `.then` and `.catch` approach.